articles/virtual-desktop/diagnostics-role-service.md

```diff
@@ -18,12 +18,11 @@ Connections that don't reach Windows Virtual Desktop won't show up in diagnostic
 
 ## Common error scenarios
 
-Error scenarios are categorized in internal to the service and external to Windows Virtual Desktop.
+The WVDErrors table tracks errors across all activity types that are tracked by the service. The column ServiceError provides an additional flag that helps you to understand whether the issue is or isn't related to the service. If the column states:
+* True: this issue can't be mitigated on your side and recommendation is to raise a support ticket under Windows Virtual desktop if you notice an unusual high occorunce. 
+* False: you will need to investigate further. The error message itself may hint the area you need to investigate next.
```

Comment:
```suggestion
- If the value is "False," then this is an issue related to the service, and is possibly something you can fix yourself. The error message can give you a clue about where to start investigating.
```

Reply:
Not quite the message I wanted to pass on. Would this work?
- If the value is "false" then this is not an issue related to the service and is possibly something you can fix yourself. The error message can give you a clue about where to start.

Reply:
If the value is "False," this is a local service issue that you can possibly fix yourself. The error message can give you a clue about where to start.

Reply:
Not quite :)! The message shouldn't be it is a service issue. It might be a misconfiguration on something which is not related to the service as all. The problem is we might see only the symptom  but can't give further guidance. The purpose of the "false" flag is to indicate that.